src/widget/OktaSignIn.js

```diff
@@ -200,7 +200,8 @@ var OktaSignIn = (function () {
 
     var OktaAuth = require('@okta/okta-auth-js/jquery');
     var Util = require('util/Util');
-    var LoginRouter = require('LoginRouter');
+    // options.useIdxPipeline defaults to false.TODO replace with Introspect API call OKTA-236343
```

Comment:
For now we are setting this option manually. Is the plan for the api to set this option value? If so, could we add some basic tests around this option?

Reply:
Once we have the introspect API we will no longer need this option. The introspect API will tell us if we should use v1 or v2.